### PR TITLE
Updated install docs and other information

### PR DESCRIPTION
**Pull request by nichollsh**
- Tidied PROTEUS setup information
- Moved troubleshooting to its own page
- Updated steps as required for the submodules
- Extra safety check in `surface_gases.py`, for when the model tries to "solve" for a gas with zero abundance

### CODE REVIEW

File: docs/source/troubleshotting.rst
Comment (lsoucasse) [inline docs/source/troubleshotting.rst:1]: Spelling mistake on the file name troubleshooting.rst

File: docs/source/installation.rst [hunk 1]
```diff
@@ -1,201 +1,89 @@
...
-                
-        * Latest tested version: *socrates_2306_trunk_r1403.tar.xz*
+        $  xcode-select --install
+   
+2. Install ``FORTRAN NetCDF`` library via the most appropriate method for you
 
-    * Interior dynamics: **SPIDER** 
-        
-        * URL: https://github.com/djbower/spider
+    .. code-block:: console
```
Comment (lsoucasse) [inline docs/source/installation.rst:20]:
```suggestion

    .. code-block:: console
```

File: docs/source/installation.rst [hunk 2]
```diff
@@ -221,246 +109,108 @@ Step-by-step (core modules)
...
-
-    .. code-block:: console
-
-        $   brew install openssl
-
-    * Follow the instructions at the end of the ``openssl`` installation (replace ``USERNAME`` with your own system username):
-
-    .. code-block:: console
+    4. Build AGNI 
+     .. code-block:: console
```
Comment (lsoucasse) [inline docs/source/installation.rst:206]:
```suggestion
        .. code-block:: console 
```

File: docs/source/installation.rst [hunk 3]
```diff
@@ -221,246 +109,108 @@ Step-by-step (core modules)
...
-        * Only **IF** ``python`` has been installed via the ``conda`` route: 
+8. Setup environment variables
 
-            .. code-block:: console
-
-                $   conda activate proteus
-        
-        * Setup the PROTEUS environment
+    * This can be done using the PROTEUS environment file
+        .. code-block:: console
```
Comment (lsoucasse) [inline docs/source/installation.rst:164]:
```suggestion

        .. code-block:: console
```

**Review by lsoucasse — CHANGES_REQUESTED** [review]
Hi @nichollsh, I like this new version of the doc. It is clearer and better structured. I made some format suggestion to homogenize the display. Also there is a spelling mistake in the file name troubleshooting.rst

**Comment by nichollsh** [comment]
Hi @lsoucasse, thanks for catching the formatting/spelling issues. I have applied your suggestions.

**Comment by lsoucasse** [comment]
Great, also when I see the build, the troubleshooting section does not appear. I think you need to update the index.rst file.

**Comment by lsoucasse** [comment]
Also in the installation.rst file, replace [:ref:`troubleshooting`] by [:doc:`troubleshooting`] with appropriate link?

**Comment by nichollsh** [comment]
@lsoucasse, I have made these changes. There's still some strange behaviour where ReadTheDocs is creating bold subtitles for some reason. Do you know why this might be?

**Review by lsoucasse — APPROVED** [review]

**Pull request merged**